app/core/src/main/java/com/fsck/k9/notification/NotificationChannelUtils.java

```diff
@@ -10,15 +10,15 @@
 import com.fsck.k9.Preferences;
 
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 
 public class NotificationChannelUtils {
     private static final String PREFIX_MESSAGES = "messages_";
     private static final String PREFIX_OTHER = "other_";
 
-    public enum Type {
-        MESSAGES, OTHER
+    public interface ChannelId {
```

Comment:
I liked the enum. It prevents callers from passing invalid values.

Reply:
Got it